crontab/crontab.go

```diff
@@ -100,11 +100,18 @@ func ParseCrontab(reader io.Reader) (*Crontab, error) {
 			continue
 		}
 
-		jobLine, err := parseJobLine(line)
+		// replace start  with so the line will parse without errors
+		jobLine, err := parseJobLine(strings.Replace(line, StartExpressionValue, "0 0 0 1 1 * *", 1))
```

Comment:
Is there a reason to call parseJobLine if we're not really trying to parse a cron expression here? If the job line starts with `@start`, how about we just split that out and return a crontab line directly, as opposed to roundtrip through `parseJobLine` with dummy data?